etc/qm/containers/containers.conf.d/qm_dropin_mount_bind_tty7.conf

```diff
@@ -0,0 +1,15 @@
+# Drop-in configuration for Podman to mount bind /dev/tty7 from host to container
+#
+# /dev/tty7 is typically the virtual terminal associated
+# with the graphical user interface (GUI) on Linux systems.
+# It is where the X server or the Wayland display server usually runs,
+# handling the graphical display, input, and windowing environment.
+# When you start a graphical session (such as GNOME, KDE, etc.),
+# it usually runs on this virtual console.
+[containers]
+mounts = [
+	"type=bind,source=/dev/tty7,target=/dev/tty7:rw"
```

Comment:
Since it is already set as a device a few lines below, the `mounts`  section could be omitted I think.

Reply:
you are right!